uaclient/entitlements/repo.py

```diff
@@ -64,15 +66,17 @@ def enable(self):
         try:
             util.subp(['apt-get', 'update'], capture=True)
             if self.packages:
-                print('Installing {title} packages'
-                      ' (this may take a while)'.format(title=self.title))
-                util.subp(['apt-get', 'install'] + self.packages)
+                print(
+                    'Installing {title} packages ...'.format(title=self.title))
```

Comment:
Driveby commit here to make sure we capture=True config changes like changing packages in /var/log/ubuntu-advantage.log instead of just printed to the console.